packages/server/lambdas/database/check-user-right.ts

```diff
@@ -22,15 +22,26 @@ export async function handleRequest(event: ALBEvent): Promise<ALBResult> {
   const userRight = {
     canRead: false,
     canWrite: false,
+    isAdmin: false,
   };
 
   try {
+    const user = await getUser(event);
+    if (isAdmin(user)) {
+      return {
+        statusCode: 200,
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ canRead: true, canWrite: true, isAdmin: true }),
+      };
+    }
+
     const params = event.queryStringParameters;
     // only listed parameters are accepted
     validateQueryParameters(params, ['category']);
     const category = params?.category;
 
-    const user = await getUser(event);
     log.info(user, `Checking user write permission for page ${category}`);
```

Comment:
Isn't user unreachable here since `const user = await getUser(event);` is now inside the trycatch block?